doc/nrf/known_issues.rst

```diff
@@ -831,6 +831,14 @@ SoftDevice Controller
 
 .. rst-class:: v1-5-0
 
+DRGN-15465: Corrupted advertising data when :option:`CONFIG_BT_EXT_ADV` is set
+  Setting scan response data using legacy HCI commands on a build with extended advertising support corrupts parts of the advertising data.
+  Zephyr sets the complete local name in the scan response data by default, and the corrupted advertising data may confuse some scanners causing the device to not be visible.
+
+  **Workaround:** Upgrade to v1.5.1 (once available) or use the master branch.
```

Comment:
Workarounds should be a way to fix the problem on the version that is affected.
If there is no known workaround then don't include one.

The one below shouldn't have been worded like this either.

Reply:
Agree that this is not a suitable workaround here (but it's okay for the issue below, since that is an issue for one version only and the only workaround is "do not use this version").

For this issue: Can you set the scan response data in some other way? Or configure Zephyr differently (since the local name is set "by default")?